7.30 Release Notes (#3983)
astronomer/docs: astro/release-notes.md

<!-- ALL LINKS TO INTERNAL DOCS MUST BE COMPLETE URLS INCLUDING HTTPS. Otherwise the links will break in RSS. -->

## July 30, 2024

> ### New alerts for Deployment health incidents

Author: @niustephanie  - we have release notes split from feature branches. Here's the release notes for the health incidents alerting.

Reviewer: @paolaperaza thoughts on releasing this feature as "New alerts for Deployment health incidents" vs "Deployment Health Alerts"? the second sounds more like a new product offering / big feature, whereas the first is clearer and more self-explanatory. I don't feel strongly but wanted a gut check

Author: Conventionally, we have typically used the headers be explanatory and then the feature names mentioned in the text. I added some bold to make that more obvious, but I'm absolutely open to changing this header. Here's a good example about Universal Metrics Exporter (https://www.astronomer.io/docs/astro/release-notes#export-metrics-about-your-astro-deployments-to-observability-tools)

Reviewer: Ah I see! Happy to keep the title as-is then 👍

<HostedBadge/>

:::privatepreview
:::

> This release introduces four new alert types that correspond to four of the existing [Deployment health incident types](https://www.astronomer.io/docs/astro/deployment-health-incidents#deployment-incidents). These new deployment health alerts allows astro to proactively notify you when deployment health issues arise.

Reviewer 2: ```suggestion
This release introduces four new alert types that correspond to four of the existing [Deployment health incident types](https://www.astronomer.io/docs/astro/deployment-health-incidents#deployment-incidents). These new deployment health alerts allows Astro to proactively notify you when deployment health issues arise.
```

> This release introduces four new alert types that correspond to four of the existing [Deployment health incident types](https://www.astronomer.io/docs/astro/deployment-health-incidents#deployment-incidents). These new deployment health alerts allows Astro to proactively notify you when deployment health issues arise.

Author: ```suggestion
This release introduces four new alert types that correspond to four of the existing [Deployment health incident types](https://www.astronomer.io/docs/astro/deployment-health-incidents#deployment-incidents). These new Deployment health alerts allows Astro to proactively notify you when Deployment health issues arise.
```

> This release introduces four new alert types that correspond to four of the existing [Deployment health incident types](https://www.astronomer.io/docs/astro/deployment-health-incidents#deployment-incidents). These new **Deployment Health Alerts** allows Astro to proactively notify you when Deployment health issues arise.

Reviewer 3: ```suggestion
This release introduces four new alert types that correspond to four of the existing [Deployment health incident types](https://www.astronomer.io/docs/astro/deployment-health-incidents#deployment-incidents). These new **Deployment Health Alerts** allow Astro to proactively notify you when Deployment health issues arise.
```

> These alerts:

Reviewer: These make sense (and are a pretty direct translation of my initial Github issue, thanks!). On second read, I think these three value props could be even more user-focused, stating what the **user** can do now vs. what the **alerts** can do. Thoughts on this?

```
Using deployment health alerts, you can:
- Improve alerting coverage beyond DAG and Task failures to address infrastructure-level incidents that are otherwise hard to monitor
- Proactively monitor deployment health and take immediate remediation actions through email, Slack, and PagerDuty to lower mean time to resolution
- Share deployment health visibility across teams by notifying the right people at the right time
```
I'm open to the wording of these specific line items being revised!

> - Use Astro architecture to identify infrastructure-level incidents that are otherwise hard to monitor

Reviewer 2: ```suggestion
- Use Astro architecture to identify infrastructure-level incidents that are otherwise hard to monitor.
```

> - Gives you granular information about your Deployment's performance, with more detail than **Healthy** or **Unhealthy**.

Reviewer 2: ```suggestion
- Give you granular information about your Deployment's performance, with more detail than **Healthy** or **Unhealthy**.
```

> - Share Deployment health visibility across teams

Reviewer 3: ```suggestion
- Share Deployment health visibility across teams.
```

> - Incorporates Astro's suggestions about Deployment health into your existing alerting workflows, including email, Slack, Pagerduty, to allow you to respond faster.

Reviewer 2: ```suggestion
- Incorporate Astro's suggestions about Deployment health into your existing alerting workflows, including email, Slack, and Pagerduty, to allow you to respond faster.
```

> - Use Astro architecture to identify infrastructure-level incidents that are otherwise hard to monitor.
> - Give you granular information about your Deployment's performance, with more detail than **Healthy** or **Unhealthy**.
> - Incorporate Astro's suggestions about Deployment health into your existing alerting workflows, including email, Slack, and Pagerduty, to allow you to respond faster.

Author: ```suggestion
Using deployment health alerts, you can:
- Improve alerting coverage beyond DAG and task failures to address infrastructure-level incidents that are otherwise difficult to monitor.
- Proactively monitor Deployment health and take immediate remediation actions through email, Slack, and PagerDuty to reduce average time to resolution.
- Share Deployment health visibility across teams
```

See [ADD LINK] for more information.

### Additional improvements

> - When you link directly to your Astro or Airflow UI, any link previews successfully have improved visuals and now include metadata information.

Reviewer 2: Is there a word missing here?

> - When you link directly to your Astro or Airflow UI, any link previews successfully have improved visuals and now include metadata information.

Author: ```suggestion
- Now, when you link directly to your Astro or Airflow UI, any link previews successfully have improved visuals and now include metadata information.
```

- Improved the Astro Alerts UI to streamline creating notification channels and alerts across DAGs and Deployments. See [Astro Alerts](https://www.astronomer.io/docs/astro/alerts) for setup information.

> - Added an example Universal Metrics Export dashboard configuration file for Grafana Cloud. See [Export metrics from Astro](https://www.astronomer.io/docs/astro/export-metrics) for setup instructions and the [Grafana Cloud](https://www.astronomer.io/docs/astro/export-metrics#grafana-example) configuration example.

Reviewer 2: ```suggestion
- Added an example Universal Metrics Export dashboard configuration file for Grafana Cloud. See [Export metrics from Astro](https://www.astronomer.io/docs/astro/export-metrics) for setup instructions and [Grafana example](https://www.astronomer.io/docs/astro/export-metrics#grafana-example) for a configuration example.
```

> - Organization Dashboards are now generally available to Enterprise tier customers. See [View Organization Dashboards](https://www.astronomer.io/docs/astro/organization-dashboard) for more information.

Reviewer 2: ```suggestion
- Organization Dashboards are now generally available to Enterprise-tier customers. See [View Organization Dashboards](https://www.astronomer.io/docs/astro/organization-dashboard) for more information.
```

### Bug fixes

> - Fixed an issue where if you linked directly to your Astro or Airflow UI, sometimes the link preview would show a 404 error. Now, link previews successfully resolve and include some metadata information.

Author: Move to Additional improvement

<!-- ALL LINKS TO INTERNAL DOCS MUST BE COMPLETE URLS INCLUDING HTTPS. Otherwise the links will break in RSS. -->

## July 30, 2024

### New alerts for Deployment health incidents

<HostedBadge/>

:::privatepreview
:::

> This release introduces four new alert types that correspond to four of the existing [Deployment health incident types](https://www.astronomer.io/docs/astro/deployment-health-incidents#deployment-incidents). These new Deployment health alerts allows Astro to proactively notify you when Deployment health issues arise.

Author: ```suggestion
This release introduces four new alert types that correspond to four of the existing [Deployment health incident types](https://www.astronomer.io/docs/astro/deployment-health-incidents#deployment-incidents). These new **Deployment Health Alerts** allows Astro to proactively notify you when Deployment health issues arise.
```

> These alerts:

Reviewer: "these alerts" should be deleted

> These alerts:

Author: ```suggestion
```

Using Deployment health alerts, you can:
- Improve alerting coverage beyond DAG and task failures to address infrastructure-level incidents that are otherwise difficult to monitor.

> - Proactively monitor Deployment health and take immediate remediation actions through email, Slack, and PagerDuty to reduce average time to resolution.

Reviewer: actually- flagging "average time to resolution" language.

> - Proactively monitor Deployment health and take immediate remediation actions through email, Slack, and PagerDuty to reduce average time to resolution.

Author: ```suggestion
- Proactively monitor Deployment health and take immediate remediation actions through email, Slack, and PagerDuty to reduce mean time to resolution.
```

- Share Deployment health visibility across teams

> See [ADD LINK] for more information.

Author: ```suggestion
See [Astro Alerts: Trigger Types](https://www.astronomer.io/docs/astro/alerts#trigger-types) and [Deployment Health Incidents](https://www.astronomer.io/docs/astro/deployment-health-incidents) for more information.
```
